### DISCUSSION TITLE
Help Me Contribute

### DISCUSSION THREAD

**Discussion by abrombo** [sympy/sympy#24064] [General]
I followed the instructions in https://github.com/sympy/sympy/wiki/Development-workflow doing

git config --global user.name "Alan Bromborsky"
git config --global user.email "abrombo999@gmail.com"
git clone git://git.kernel.org/pub/scm/git/git.git
cp git/contrib/completion/git-completion.bash ~/.git-completion.sh
ssh-keygen -t ed25519 -C "abrombo999@gmail.com"
cat ~/.ssh/id_ed25519.pub
sudo apt-get install python3-pip python3-dev python3-setuptools apache2 -y
pip3 install --upgrade pip
pip3 install sphinx
git clone https://github.com/sympy/sympy.git
cd sympy
git remote add github git@github.com:Uttam-Singhh/sympy.git
git checkout -b brombotronics

and I can no longer find sympy on my computer to add to.  Where is it?  I did not set up a virtual environment.  I do not plan on modifying any existing sympy code only adding my modules.

**Comment by asmeurer** [comment]
If you entered those commands then the sympy git repo should be in whatever directory you started in. `git clone` will create a new directory `sympy` in the directory you are in, and the `cd sympy` line after that will change to that directory. 

Note also that the commands at the beginning are only necessary to run once. Once you create an SSH key and upload it to GitHub you do not need to do so again.
